sigle/src/modules/publicHome/components/PublicStoryItem.tsx

```diff
@@ -80,38 +54,173 @@ export const PublicStoryItem = ({ username, story, settings }: Props) => {
+                    mb: story.featured ? '$5' : 'none',
+                  },
+                }}
+              >
+                <StoryImage
+                  css={{
+                    width: story.featured ? 420 : 80,
+                    height: story.featured ? 214 : 58,
+
+                    '@md': {
+                      width: story.featured ? 826 : 180,
+                      height: story.featured ? 420 : 130,
+                    },
+                  }}
+                  data-testid="story-cover-image"
+                  src={story.coverImage}
+                />
+              </Box>
+            </Box>
+          </Link>
+        )}
+        <Flex direction="column" css={{ flex: 1 }}>
+          <Box
+            css={{
+              '@md': {
+                mb: '$5',
+              },
+            }}
           >
-            <Heading size="xl" data-testid="story-title">
-              {story.title}
-            </Heading>
-            <Text size="sm" data-testid="story-date">
-              {format(story.createdAt, 'dd MMMM yyyy')}
-            </Text>
-            <Text
-              css={{ mt: '$4', color: '$gray10' }}
-              size="sm"
-              data-testid="story-content"
+            <Flex
+              justify="between"
+              css={{ gap: '$5', mb: '$1', '@md': { mb: '$2' } }}
             >
-              {story.content}
-            </Text>
-          </StoryContainerContent>
-        </StoryContainer>
-      </Link>
-    </Container>
+              <Link
+                href="/stories/[storyId]"
+                as={`/stories/${story.id}`}
+                passHref
+              >
+                <a>
+                  <Typography
+                    size={{
+                      '@initial': 'subheading',
+                      '@md': 'h4',
+                    }}
+                    css={{
+                      alignSelf: 'center',
+                      fontWeight: 600,
+                      overflow: 'hidden',
+                      display: '-webkit-box',
+                      '-webkit-line-clamp': 3,
+                      '-webkit-box-orient': 'vertical',
+                      typographyOverflow: 'ellipsis',
+
+                      '@md': {
+                        '-webkit-line-clamp': 2,
+                      },
+                    }}
+                    as="h4"
+                    data-testid="story-title"
+                  >
+                    {story.title}
+                  </Typography>
+                </a>
+              </Link>
+            </Flex>
+            <Link
+              href="/stories/[storyId]"
+              as={`/stories/${story.id}`}
+              passHref
+            >
+              <Typography
+                data-testid="story-date"
+                css={{ color: '$gray9' }}
+                size="subparagraph"
+                as="a"
+              >
+                {format(story.createdAt, 'MMMM dd, yyyy ')}
+                <Box
```

Comment:
@QuentinSaubadu can we remove this part? Many stories only have the day and not the time